hack/e2e-common.sh

```diff
@@ -87,6 +100,12 @@ function save_hive_logs() {
     if oc logs -n $n deployment/$d > $tmpf; then
       mv $tmpf "${ARTIFACT_DIR}/${d}.log"
     fi
+    # If deployments' pods didn't start, the above won't produce logs; but these statuses may hint why.
```

Comment:
looks to me like these particular changes are already included in https://github.com/openshift/hive/pull/1781

Reply:
Yeah, so this is a limitation of GH that you can't stack PRs without the Nth one including all the commits from <N. It should rebase on merge, since the other one landed already.